tests/test_index.py

```diff
@@ -0,0 +1,62 @@
+import io
+from pathlib import Path
+from unittest.mock import Mock
+
+import pytest
+
+from dissect.thumbcache.exceptions import NotAnIndexFileError
+from dissect.thumbcache.index import IndexEntry, ThumbnailIndex, ThumbnailType
+
+
+def test_index():
+    ThumbnailIndex(file=Mock)
+
+
+@pytest.mark.parametrize(
+    "path, expected_type",
+    [
+        (Path("data/windows_vista/thumbcache_idx.db"), ThumbnailType.WINDOWS_VISTA),
+        (Path("data/windows_7/thumbcache_idx.db"), ThumbnailType.WINDOWS_7),
+        (Path("data/windows_81/thumbcache_idx.db"), ThumbnailType.WINDOWS_81),
+        (Path("data/windows_10/thumbcache_idx.db"), ThumbnailType.WINDOWS_10),
+        (Path("data/windows_11/thumbcache_idx.db"), ThumbnailType.WINDOWS_10),
+    ],
+)
+def test_index_type(
+    path: Path,
+    expected_type: ThumbnailType,
+) -> None:
```

Comment:
In the other code if've seen you not specifying the return type when None. For consistency, remove the one here :)
```suggestion
):
```